umm-spec-lib/src/cmr/umm_spec/umm_g/data_granule.clj

```diff
@@ -0,0 +1,35 @@
+(ns cmr.umm-spec.umm-g.data-granule
+  "Contains functions for parsing UMM-G JSON DataGranule into umm-lib granule model
+   DataGranule and generating UMM-G JSON DataGranule from umm-lib granule model DataGranule."
+  (:require
+   [cmr.umm.umm-granule :as g])
+  (:import cmr.umm.umm_granule.UmmGranule))
+
+(defn umm-g-data-granule->DataGranule
+  "Returns the umm-lib granule model DataGranule from the given UMM-G DataGranule."
+  [data-granule]
+  (when data-granule
+    (g/map->DataGranule
+      (let [{:keys [Identifiers DayNightFlag ProductionDateTime
+                    ArchiveAndDistributionInformation]} data-granule]
+        {:day-night DayNightFlag
```

Comment:
UMM-G uses different case for DayNightFlag from umm-lib granule model which is basely what is ECHO10, this function should handle the mapping correctly.

Reply:
oh. I saw you comment on the top of the PR stating that this is intentional. I guess hoping Erich would modify the umm-g schema to change the DayNightFlag Enums to match ECHO10. 

Given how easy it is to just code in the mapping, I think we should just fix it right now rather than letting loose ends hanging in the code. We can always remove the mapping code if we decide to modify the casing later.

Reply:
The same can be said for leaving it and re visiting when Erich returns.  Now I do double to work to reverse it, only to have the chance of reversing it again.